csharp/ql/src/utils/modelgenerator/internal/CaptureModels.qll

```diff
@@ -4,11 +4,20 @@
  */
 
 private import CaptureModelsSpecific
+private import CaptureModelsPrinting
 
 class DataFlowTargetApi extends TargetApiSpecific {
   DataFlowTargetApi() { isRelevantForDataFlowModels(this) }
 }
 
+private module Printing implements PrintingSig {
+  class Api = DataFlowTargetApi;
+
+  string getProvenance() { result = "df-generated" }
+}
+
+module ModelPrinting = PrintingImpl<Printing>;
```

Comment:
private?

Reply:
It is used in a file importing `CaptureModels.qll`.